site/docs/components/link/usage.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Using the Link component
+
+A standalone link is typically displayed in a body text style in the primary variant. If you want to display the link with less prominence, such as when linking to secondary information, you can display it in the secondary variant.
+
+An inline link should always use the same text formatting as the text that it is displayed with.
+
+#### When to use Link
+
+- To provide navigation to a page on the same or different site.
+- To link to documents, email addresses or phone numbers.
+- To send users to a specific section on the same page.
+- To provide users with more detailed information or assistance e.g., terms and conditions, help sections or contact information.
+
+#### When not to use Link
+
+To trigger an action, such as submitting a form or opening a Dialog. Instead, use [Button](../../button).
```

Comment:
```suggestion
To trigger an action, such as submitting a form or opening a dialog. Instead, use [Button](../../button).
```
unless its meant to refer to the Dialog component? in which case we'll also need to link the component

Reply:
Ultimately I think it will refer to the component but for now it's safer to make it lowercase